Assets/MixedRealityToolkit/Inspectors/ServiceInspectors/HandJointServiceInspector.cs

```diff
@@ -0,0 +1,271 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using System.Collections.Generic;
+using System;
+using UnityEditor;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
+{
+    [MixedRealityServiceInspector(typeof(HandJointService))]
+    public class HandJointServiceInspector : BaseMixedRealityServiceInspector
+    {
+        private const string ShowHandPreviewInSceneViewKey = "MRTK_HandJointServiceInspector_ShowHandPreviewInSceneViewKey";
+        private const string ShowHandJointFoldoutKey = "MRTK_HandJointServiceInspector_ShowHandJointFoldoutKey";
+        private const string ShowHandJointKeyPrefix = "MRTK_HandJointServiceInspector_ShowHandJointKeyPrefixKey_";
+        private static bool ShowHandPreviewInSceneView = false;
+        private static bool ShowHandJointFoldout = false;
+
+        private const float previewJointSize = 0.02f;
+        private static readonly Color previewJointColor = new Color(0.5f, 0.1f, 0.6f, 0.75f);
+        private static readonly Color enabledColor = GUI.backgroundColor;
+        private static readonly Color disabledColor = Color.Lerp(enabledColor, Color.clear, 0.5f);
```

Comment:
> private static readonly Color disabledColor = Color.Lerp(enabledColor, Color.clear, 0.5f); [](start = 7, length = 91)

Thoughts on having a Colors.cs where these types of constants can go?

I see disabledColor copy pasted across a few different service inspector implementations.

Reply:
I agree this would be nice. @chrisfromwork has a related proposal open here: #3908